AlertaDengue/dados/views.py

```diff
@@ -205,7 +205,7 @@ def get_context_data(self, **kwargs):
         if service == "maps":
             if service_type is None:
 
-                geo_info_path = get_static("geojson/geo_info.json")
```

Comment:
for some reason, `get_static` returns an absolute path (/static/geojson/geo_info.json) instead of relative, I replaced by `staticfiles.find()` which returns a real absoute path from root